.envrc

```diff
@@ -1,3 +1,3 @@
-export KUBEBUILDER_ASSETS=$PWD/testbin/bin
```

Comment:
Darwin is specific to Mac OS and will be nonfunctional on linux systems. It is also specific to a version, which may change.

We should probably think about how to resolve this or remove the .envrc file from the project and add a .gitignore for people to configure for themselves.

Reply:
Looks like it highlighted just one half of the diff...

Reply:
We replaced this with a call to `setup-envtest` to retrieve the appropriate value.